Shared/Realm.Shared/Linq/IRealmResults.cs

```diff
@@ -16,10 +16,94 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+
 namespace Realms
 {
     internal interface IRealmResults
     {
         Schema.ObjectSchema ObjectSchema { get; }
```

Comment:
Do we want to expose it publicly? I'm fairly conservative about exposing stuff that people are unlikely to need.